packages/builder/src/stores/portal/licensing.js

```diff
@@ -198,11 +198,11 @@ export const createLicensingStore = () => {
         const userQuota = license.quotas.usage.static.users
         const userLimit = userQuota?.value
         const userCount = usage.usageQuota.users
-        const userLimitReached = willReachUserLimit(userCount, userLimit)
-        const userLimitExceeded = willExceedUserLimit(userCount, userLimit)
+        const userLimitReached = usersLimitReached(userCount, userLimit)
+        const userLimitExceeded = usersLimitExceeded(userCount, userLimit)
```

Comment:
What is the difference between this check:

`usersLimitReached(userCount, userLimit)`

And this other one in the same file:

`usersLimitReached(userCount, get(store).userLimit)`

Why do we need to check with static value and the value in the store? Do we still needing this? What cases?

Reply:
the main idea is that we expose some functions from the store:

`willReachUserLimit`
`willExceedUserLimit` 

These both take an argument of a user count. This is compared against the user limit, and the appropriate return value given. 

These are used by the components that create users
- `AddUserModal`
- `ImportUsersModal`

Where these modals can supply a new count of what the total will be, so that they can pre-emptively show warnings / disable inputs. 

Also in the store we have the usage of the above functions, for creating constants for 
- `userLimitReached`
- `userLimitExceeded`

Rather than being pre-emptive calculations, these are real indicators if this user limit has been reached with the real users count. These are used for
- Showing the upgrade modals for adding new users
- Showing the user limit banner

The functions just default to grabbing the current user limit from the store
```ts
    willReachUserLimit: userCount => {
      return willReachUserLimit(userCount, get(store).userLimit)
    },
```
while the creation of the constants supply the limit they have already have access to as the store is being populated for the first time
```ts
const userLimitReached = willReachUserLimit(userCount, userLimit)
```